js/src/main/scala/crystal/react/ContextProvider.scala

```diff
@@ -12,17 +11,13 @@ object ContextProvider {
   @inline def apply[F[_]] = new Apply[F]
 
   class Apply[F[_]] {
-    def apply[C](ctx: Ctx[F, C], initCtx: C)(
-      render:         VdomNode
-    )(implicit
-      reusabilityC:   Reusability[C],
-      effect:         Effect[F],
-      cs:             ContextShift[F]
-    ): StateComponent[C] =
+    def apply[C](ctx:        Ctx[F, C], initCtx:    C)(
+      render:                VdomNode
+    )(implicit reusabilityC: Reusability[C], async: Async[F]): StateComponent[C] =
       ScalaComponent
         .builder[Unit]
         .initialState(initCtx)
-        .render($ => ctx.provide(ViewF.fromState[F]($))(render))
+        .render($ => ctx.provide(ViewF.fromState[F](async)($))(render))
```

Comment:
`async` should pass implicitly but for some reason it doesn't and I have to pass it explicitly